doc/user_guide/install.rst

```diff
@@ -236,41 +205,3 @@ You can create binaries for Debian/Ubuntu from the source by running the
     For this to work, the following packages must be installed in your system
     python-stdeb, debhelper, dpkg-dev and python-argparser are required.
 
-
-.. _install-dependencies:
-
-Installing the required libraries
----------------------------------
-
-
-In addition to the libraries that are automatically installed when installing
-HyperSpy using ``pip`` (see :ref:`install-with-python-installers`).
-
-In case some of the required libraries are not automatically installed when
-installing from source in a conda environment, these can be obtained beforehand
-by installing and removing hyperspy from that environment;
-
-.. code-block:: bash
-
-    $ conda install hyperspy
-    $ conda remove hyperspy
-    $ pip install -e ./
-
-.. _known-issues:
-
-Known issues
-------------
-
-Windows
-^^^^^^^
-
-* If HyperSpy fails to start in Windows try installing the Microsoft Visual
-  before reporting a bug.
-
-
-* If HyperSpy raises a MemoryError exception:
-
-  * Install the 64bit version if you're using the 32bit one and you are running
-    HyperSpy in a 64bit system.
-  * Increase the available RAM by closing other applications or physically
-    adding more RAM to your computer.
```

Comment:
Remove this section because this unclear what the `Microsoft Visual` is and normally `pip` should provide a more appropriate message than this comment here.
`MemoryError` is not really relevant here, particularly since most current binaries distribution are 64 bits.